src/main/java/com/helloworld/greet/Students.java

```diff
@@ -0,0 +1,26 @@
+package com.helloworld.greet;
+
+public class Students {
```

Comment:
パスは複数形でよいのですが、クラス名は単数形でよいです。
```suggestion
public class Student {
```

Reply:
パスはRESTの考えにのっとる
クラス名はJavaの考えにのっとる
というイメージです。

Reply:
クラス名を単数形に変更しました。
b0f8d50b60bcdc8541475e56ea2cf55ef3a9e20e

よく単数形、複数形で混乱しているので
ご指摘いただいた点を含めて改めて落とし込みたいと思います。